pkg/logchannel/logchannel.go

```diff
@@ -0,0 +1,49 @@
+package logchannel
```

Comment:
vendoring in golang is [more-or-less standardized](https://blog.gopheracademy.com/advent-2015/vendor-folder/)

I use the [govendor](https://github.com/kardianos/govendor) utility in this project to manage the `vendor` directory

Can you:
* install the `govendor` utility on your computer
* run `govendor get github.com/arkan/bastion/pkg/logchannel && govendor add github.com/arkan/bastion/pkg/logchannel`
* remove this file
* replace the `import` path in `bastionsession.go` (`moul/sshportal` -> `arkan/bastion`)

Reply:
ok done. Thanks, for the lesson :)